docs/troubleshooting-guide/running.md

```diff
@@ -26,3 +26,7 @@ If the Metabase instance starts and runs for a significant amount of time before
     java -Xmx2g -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/path/to/a/directory -jar metabase-jar
 
 The `-XX:HeapDumpPath` flag is optional, with the current directory being the default. When an `OutOfMemoryError` occurs, it will dump an `hprof` file to the directory specified. These can be large (i.e. the size of the `-Xmx` argument) so ensure your disk has enough space. These `hprof` files can be read with many different tools, such as `jhat` included with the JDK or the [Eclipse Memory Analyzer Tool](https://www.eclipse.org/mat/).
+
+### Metabase can't read or write from a file or directory (IOError)
+
+Check the issue [here](docker.md)
```

Comment:
I suggest not using "here" as the link text (see [this old advice](https://www.w3.org/QA/Tips/noClickHere)).
If anything, "here" isn't a good search-able keyword.